### ISSUE TITLE
Is there a possibility to keep a changelog?

### ISSUE THREAD

**Issue by luzpaz** [manuel-serrano/bigloo#123]
See https://keepachangelog.com/

The reason is that it's very hard to parse the repo to find the latest version.

**Comment by manuel-serrano** [comment]
Yes, that would a good idea. At least to have a global file that simply document the main changes and mail releases. I will add one starting with release 4.6a. Thanks for the suggestion.
